XenAdmin/TabPages/HomePage.Designer.cs

```diff
@@ -34,16 +34,19 @@ private void InitializeComponent()
             // 
             // webBrowser
             // 
+            this.webBrowser.AllowWebBrowserDrop = false;
             resources.ApplyResources(this.webBrowser, "webBrowser");
+            this.webBrowser.IsWebBrowserContextMenuEnabled = false;
             this.webBrowser.Name = "webBrowser";
+            this.webBrowser.ScriptErrorsSuppressed = true;

```

Comment:
Not sure why but the scrollbar is now visible, `ScrollBarsEnabled = false` addressed this issue

Reply:
At the minimum main window size and the default position of the vertical splitter the home page should look ok - at least I don't see scrollbars in XC (there do exist in CHC though because that homepage is bigger; the old size will be restored with CP-41433). Unfortunately we can't disable the scrollbars completely because if the user moves the splitter to the right the home page will be truncated.